advanced/Scripts/database_migration/gravity-db.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090
+
+# Pi-hole: A black hole for Internet advertisements
+# (c) 2019 Pi-hole, LLC (https://pi-hole.net)
+# Network-wide ad blocking via your own hardware.
+#
+# Updates gravity.db database
+#
+# This file is copyright under the latest version of the EUPL.
+# Please see LICENSE file for your rights under this license.
+
+upgrade_gravityDB(){
+	local database auditFile version
+	database="${1}"
+	auditFile="${2}"
```

Comment:
This function should be generic enough for all migrations we do. The audit file location should therefore be in this file, not passed in from gravity.